### DISCUSSION TITLE
Box around face is blurred

### DISCUSSION THREAD

**Discussion by jonxq** [hacksider/Deep-Live-Cam#689] [Q&A]
Hello, when I turn ON deep live cam and sit down to webcamera is very close , there pixels and blured things around my face, but then when I sit down not so close to webcamera its normal and I dont have pixels around my face, can I fix it somehow? maybe somewhere I can change box size around face?

**Comment by hacksider** [comment]
I suggest dont sit too close on the camera then 😄 
Please be informed that the model that is in use is just 128x128 (insightface did not release the higher models).

**Reply by ineow1** [reply]
when do you think they will release higher models

**Reply by KRSHH** [reply]
They will at some point but, not anytime soon. - they are using it commercially as of now - picsi.ai

**Reply by hacksider** [reply]
I have talked to the CEO before and they don't have any plans of releasing it to the public as it is deemed too dangerous on a wrong hand.

So most probably an online version only or via API.

**Reply by KRSHH** [reply]
> I have talked to the CEO before and they don't have any plans of releasing it to the public as it is deemed too dangerous on a wrong hand.
> 
> So most probably an online version only or via API.

GPT-2 was too dangerous to release at some point... somebody else will release a larger model, if not them.
